### PR TITLE
Add Cond to Pytorch frontend Linear Algebra #14591

### LINKED ISSUE #14591: cond

### PR DESCRIPTION
**Pull request by OmarJabri7**
Added Function cond with unit test for Linear Algebra Pytorch.
close #14591

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 1]
```diff
@@ -410,6 +410,25 @@ def test_torch_eigvalsh(
     )
 
 
+@handle_frontend_test(
+    fn_tree="torch.linalg.cond",
+    aliases=["torch.cond"],
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:415]: This alias doesn't exist as well

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @OmarJabri7 

Thanks for the PR 😊
Some changes are needed check the comments below.

Thanks

**Comment by OmarJabri7** [comment]
Hello @hirwa-nshuti , I applied some changes however I still see an assertion error np.nan!= np.inf when comparing ground truth.
Can that be due to the order of my parameters ?

**Comment by OmarJabri7** [comment]
@hirwa-nshuti I think we can now:
close #14591

**Comment by OmarJabri7** [comment]
close #14591

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 2]
```diff
@@ -514,8 +555,8 @@ def test_torch_cross(
         frontend=frontend,
         test_flags=test_flags,
         fn_tree=fn_tree,
-        rtol=1e-2,
-        atol=1e-3,
+        rtol=1e-9,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:558]: Would you revert these changes

Comment (OmarJabri7) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:585]: @hirwa-nshuti  sure, I will reput them. I asserted now in the actual function.
The problem is that I want to exclude non singular matrices.
What do you think about the new changes?

Comment (OmarJabri7) [reply ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:584]: @hirwa-nshuti 
I fixed the issues, my tests are failing only on checks that are not related to my code changes at all.
What can I do in this scenario?
Can we close #14622  #14591

**Review by hirwa-nshuti — APPROVED** [review]
LGTM!
Thanks for contributing😊
